### DISCUSSION TITLE
Unstable ubuntu-22.04 runners

### DISCUSSION THREAD

**Discussion by mcorino** [actions/virtual-environments#9633] [General questions] [answered]
What is happening with the ubuntu-22.04 runners?
For the last 4 days I'm troubled with regularly failing builds on these runners because of problems in resolving package repository servers (see [here](https://github.com/mcorino/wxRuby3/actions/runs/8552748405/job/23434522414#step:12:245).
Intermittently I also have builds that do succeed but the situation is definitely not stable.

**Comment by shamil-mubarakshin** [comment, marked as answer]
Hey @mcorino,
Don't see if it is already present, but can you try adding `sudo apt-get update` to your workflow?
It is recommended to update package information before using apt

**Comment by mcorino** [comment]
I'll give that a go. Thx.
Although I have to say; I had this workflow working pretty much flawlessly for months before it started failing 4 days ago.

**Comment by mcorino** [comment]
Ok, that seems to help. Hopefully it keeps.
